app/controllers/health_controller.rb

```diff
@@ -0,0 +1,13 @@
+class HealthController < ApplicationController
+  def workers
+    status = worker_health_checker.all_healthy? ? :ok : :internal_error
+
+    render json: worker_health_checker.statuses, status: status
```

Comment:
Yes, that is totally possibly. I was thinking of making a `Response` object with all the fields and was on the fence about it. This is a good enough reason to make it, will fix now.